### PR TITLE
[build.webkit.org] Rename WinCairo-* builders to Windows-*

### PR DESCRIPTION
**Pull request by fujii**
#### 0c44fdea9a036ceaf533e8708b167b36799380db
<pre>
[build.webkit.org] Rename WinCairo-* builders to Windows-*
<a href="https://bugs.webkit.org/show_bug.cgi?id=279405">https://bugs.webkit.org/show_bug.cgi?id=279405</a>

Reviewed by Don Olmstead.

WinCairo port has been renamed to Windows port. Update post-commit
Buildbot configurations.

Renamed WinCairo-64-bit-{Release,Debug}-Build builders to
Windows-64-bit-{Release,Debug}-Build.

Removed unused WinCairo-64-bit-WKL-Release-Tests builders.

Renamed a platform name &quot;wincairo&quot; to &quot;win&quot;.

The worker names still contain &quot;wincairo&quot;. Will rename them later.

Updated bot watcher&apos;s dashboard configurations.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories.py:
(TestFactory.__init__):
(TestJSCFactory.__init__):
* Tools/CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* Tools/CISupport/build-webkit-org/loadConfig.py:
(loadBuilderConfig):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/Windows7.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/Windows8.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js:
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js:
(WebKitBuildbot):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css:
(table.queue-grid tr.platform.windows img.logo):
(table.queue-grid tr.platform.windows-10 td.logo::before): Deleted.
(table.queue-grid tr.platform.wincairo-windows-10 td.logo::before): Deleted.
(table.queue-grid tr.platform.wincairo-windows-10 img.logo): Deleted.
(table.queue-grid tr.platform.windows-10 img.logo): Deleted.
(table.queue-grid tr.platform.windows-7 img.logo): Deleted.
* Tools/CISupport/build-webkit-org/steps.py:
(CustomFlagsMixin.customBuildFlag):
(CustomFlagsMixin.appendCustomBuildFlags):
(CustomFlagsMixin.appendCustomTestingFlags):
(ShellMixin):
(CleanUpGitIndexLock.run):
(InstallWindowsDependencies):
(RunJavaScriptCoreTests.run):
(InstallWinCairoDependencies): Deleted.
* Tools/CISupport/build-webkit-org/wkbuild.py:
(_should_file_trigger_build):

Canonical link: <a href="https://commits.webkit.org/283531@main">https://commits.webkit.org/283531@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/f5e5dbef5f4ec74f192d641b16fffc4d1f678810

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/66587 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/45962 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/19208 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/70621 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/17720 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/68705 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/53761 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/17480 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/53362 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 wincairo-tests~~](https://ews-build.webkit.org/#/builders/60/builds/11948 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/69654 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/42319 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/57620 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/34027 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/38990 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/64/builds/15011 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/16074 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/60902 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/63/builds/15352 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/72323 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/10544 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/14716 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/60689 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/66267 "Passed tests") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/10576 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/57691 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/61017 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/8673 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/119/builds/2294 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/10090 "Built successfully and passed tests") | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/41769 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/42846 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/44029 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/42589 "Built successfully") | | |

### CODE REVIEW

File: Tools/CISupport/build-webkit-org/config.json
Comment (donny-dont) [inline Tools/CISupport/build-webkit-org/config.json:81]: If this change is going to require the bots to reconnect why don't we just rename the bots now?

**Review by donny-dont — APPROVED** [review]
This is fine but I feel like someone on the infrastructure team should take a look before landing.

File: Tools/CISupport/build-webkit-org/config.json
Comment (fujii) [reply Tools/CISupport/build-webkit-org/config.json:81]: I think it requires updating `password.json` on the master. I'm changing gradually.

Comment (donny-dont) [reply Tools/CISupport/build-webkit-org/config.json:81]: @ryanhaddad @briannafan is that the case with the `password.json`?

Comment (ryanhaddad) [reply Tools/CISupport/build-webkit-org/config.json:81]: Yes, this is true. We can coordinate making the changes.

Comment (fujii) [reply Tools/CISupport/build-webkit-org/config.json:81]: Thank you. I will do that in a later patch after I confirm there is no problem with this change. Worker name is less important than builder name.

**Pull request merged**
